hq/public/javascripts/app.js

```diff
@@ -66,16 +66,6 @@ $(document).ready(function() {
     });
   });
 
-  // Extra interactions for the dropdowns on the Security Groups page
```

Comment:
The security groups page is separate to the security groups lambda, but I guess no harm doing this here!

The security groups page was a SHQ display of the state of all our security groups (now duplicated [here](https://metrics.gutools.co.uk/d/ddkehpibugpogd/security-groups?orgId=1)). The security-groups Lambda was the automated real-time alerting that gets sent whenever a security group gets opened up to the world.

In retrospect, the naming of these two concepts wanted some more thought!